.doc_gen/readmes/readme.txt

```diff
@@ -0,0 +1,63 @@
+# {{ service_name }} code examples for the {{ short_sdk_name }}
+## Overview
+These examples show how to {{ code_examples_actions }} using the {{ short_sdk_name }}.
```

Comment:
Line 3 is the first in-text mention of the SDK name, so we should use the long SDK name.

Reply:
done.